config/checkstyle/checkstyle.xml

```diff
@@ -179,7 +180,7 @@
     </module>
     <module name="AbbreviationAsWordInName">
       <property name="ignoreFinal" value="false"/>
-      <property name="allowedAbbreviations" value="JNDI,UUID"/>
+      <property name="allowedAbbreviationLength" value="40"/>
```

Comment:
Using CAPS_PROPS in the Test file - better for readability.

Reply:
What does this fix?
you may as well remove this full, 40 defeats the purpose of it